src/Contract/Operation/Dropable.php

```diff
@@ -18,7 +18,9 @@
 interface Dropable
 {
     /**
-     * Skip the n items of a collection.
+     * Drop the `n` first items of the collection.
+     *
+     * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#drop
```

Comment:
@drupol I found this operation quite weird, especially because it takes variadic parameters and then adds them up.

```PHP
Collection::fromIterable([1, 2, 3, 4, 5])->drop(1, 4); 
// result -> []
// equivalent to:
Collection::fromIterable([1, 2, 3, 4, 5])->drop(5); 
```

In this example it drops the first 1+4=5 elements of the collection. As a user I wouldn't expect this, maybe I would expect it to drop keys 1 and 4.

Is there a use case for variadic parameters here? I would propose to remove that and only allow a single count, which is the number of items to drop.

Reply:
I agree with that point, this would be a BC break, but I'm definitely ok with this in v5.